spec/stack.js

```diff
@@ -13,10 +13,62 @@ describe('Stack', () => {
 
   context('push()', () => {
     it('pushes an element to the top of the stack.', () => {
-      const myStack = new Stack()
+      const stack = new Stack()
 
-      expect(() => myStack.push('foo'))
-        .to.alter(() => myStack.length(), { from: 0, to: 1 })
+      expect(() => stack.push('foo'))
+        .to.alter(() => stack.length(), { from: 0, to: 1 })
+    })
+
+    it('pushes an element to the top of the stack when there is preexisting data', () => {
+      const stack = new Stack()
+      stack.push('alice')
+      stack.push('bob')
+
+      expect(() => stack.push('foo'))
+        .to.alter(() => stack.peek(), {from: 'bob', to: 'foo'})
+    })
+  })
+
+  context('pop()', () => {
+    it('takes an element from the top of the stack.', () => {
+      const stack = new Stack()
+      stack.push('bob')
+      stack.push('alice')
+
+      expect(() => stack.pop())
+        .to.alter(() => stack.length(), { from: 2, to: 1 })
```

Comment:
I would also add another test to check the contents of the stack after calling `pop`. This would test if the content is removed from the right direction.